Makefile

```diff
@@ -13,3 +16,26 @@ benchmark_core:
 
 benchmark_aux:
 	bash ./benchmark/benchmark_aux.sh
+
+tests_common_gpu:
+	python -m pytest tests/test_* $(if $(IS_GITHUB_CI),--report-log "common_tests.log",)
+
+slow_sft_tests:
+	python -m pytest tests/slow/test_sft_slow.py $(if $(IS_GITHUB_CI),--report-log "sft_slow.log",)
+
+slow_dpo_tests:
+	python -m pytest tests/slow/test_dpo_slow.py $(if $(IS_GITHUB_CI),--report-log "dpo_slow.log",)
+
+run_sft_examples:
```

Comment:
This is a makefile command that loops over all accelerate configs inside `ACCELERATE_CONFIG_PATH` and run SFT / DPO script with these accelerate configs